release_notes.md

```diff
@@ -0,0 +1,58 @@
+# What's New
+
+- This solution pack is now **Certified**
```

Comment:
It was certified earlier too, no?

Reply:
No, the Earlier GDPR SP version was not certified.